File: SPRINGBOOT/demo/src/main/java/com/example/classes/Ritirante.java
package com.example.classes;

public class Ritirante {
    private String targa;
    private int idAutotrasportatore;
    private String autotrasportatore;
    private int id;

    public Ritirante(int id, String targa, int idAutotrasportatore, String autotrasportatore) {
        this.targa = targa;
        this.idAutotrasportatore = idAutotrasportatore;
        this.autotrasportatore = autotrasportatore;
        this.id = id;
    }

    public String getTarga() {
        return targa;
    }

    public int getIdAutotrasportatore() {
        return idAutotrasportatore;
    }

    public String getAutotrasportatore() {
        return autotrasportatore;
    }

    public int getId() {
        return id;
    }
}

File: PHP/classi/Ritirante.php
<?php
    class Ritirante{
        private $idAutotraportatore;
        private $autotrasportatore;
        private $targa;
        private $id;

        public function __construct($id, $idAutotraportatore, $autotrasportatore, $targa) {
            $this->id = $id;
            $this->idAutotraportatore = $idAutotraportatore;
            $this->autotrasportatore = $autotrasportatore;
            $this->targa = $targa;
        }

        public function getIdAutotraportatore() {
            return $this->idAutotraportatore;
        }

        public function getAutotrasportatore() {
            return $this->autotrasportatore;
        }

        public function getTarga() {
            return $this->targa;
        }

        public function getId() {
            return $this->id;
        }
    }
?>
